### ISSUE TITLE
Use git-describe for versioning dev builds.

### ISSUE THREAD

**Issue by RealKindOne** [NationalSecurityAgency/ghidra#2006]
Use git-describe for versioning, this will show the commit ghidra was built on. Or just include the git commit in the version About window. 
```
user@laptop:~/build/ghidra$ git describe
Ghidra_9.1.2_build-1012-gbdbbca30
```
Current output:

Help -> About Ghidra
```
Version 9.2
Build DEV
2020-Jun-18 0836 EDT
```

This can be confusing when someone compiles from git and files a bug report. Is this the current git head/master or was that build from a git repo that has not been updated in 2 months?

Having the git commit in the version number helps when trying to search for when the bug, regression, issue, or whatever was introduced when using ```git bisect``` for example.

Suggested output:

Help -> About Ghidra
```
Version 9.2
Build DEV git bdbbca30
2020-Jun-18 0836 EDT
```

**Comment by ghidra1** [comment]
The git commit information for "official" builds is contained within the file Ghidra/application.properties.   For local development builds we may not be adding this to the properties file but we could look into doing so.

**Comment by ghizard** [comment]
Closed by merge of branch faa9518
